pkg/url/url.go

```diff
@@ -27,7 +28,14 @@ func Delete(client *occlient.Client, name string) error {
 func Create(client *occlient.Client, componentName, applicationName string) (*URL, error) {
 	labels := componentlabels.GetLabels(componentName, applicationName, false)
 
-	route, err := client.CreateRoute(componentName, labels)
+	// Namespace the component
+	namespacedOpenShiftObject, err := util.NamespaceOpenShiftObject(componentName, applicationName)
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to create namespaced name")
+	}
+
+	// TODO: Make sure NAME is only componentName not componentName-applicationName
```

Comment:
why?

Reply:
@kadel Was my TODO since the URL is failing, since as of right now, it'll create a URL with name "applicationName-componentName" when it should just be "componentName"